### DISCUSSION TITLE
Where has StartSharp repository gone ?

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#5301] [Help]
hey there, i cannot find StartSharp Repository with change of serenity.is to serenity-is

**Comment by edwardch** [comment]
It's still there, under volkan's name (from my side). I wonder if its a DNS (or other) thing?

I see they have been very busy on Serenity lately, especially on .NET Core 5 updates (see other branches in this repo). 

Very exciting stuff. I'm looking forward to the future and direction of Serenity/Serene/StartSharp!

**Comment by VictorTomaili** [comment]
I convert this issue to discussion. @kilroyFR it's still same url. nothing changed yet.

+Thanks @edwardch

**Comment by kilroyFR** [comment]
ok found it. Thanks Edward
